lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -1643,6 +1653,12 @@ LowererMDArch::GeneratePrologueStackProbe(IR::Instr *entryInstr, IntConstType fr
     // $done:
     //
 
+    if (this->m_func->IsTrueLeaf() &&
+        frameSize - Js::Constants::MinStackJIT < Js::Constants::MaxStackSizeForNoProbe)
+    {
+        return;
+    }
```

Comment:
Please add comment on why we decide to skip probing stack. Also, you can consider moving this code above the existing comments - because the existing comment is for the case where we generate ProbeCurrentStack